GVFS/GVFS.Common/GVFSEnlistmentStatistics.cs

```diff
@@ -0,0 +1,189 @@
+﻿using GVFS.Common.Database;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace GVFS.Common
+{
+    public class GVFSEnlistmentStatistics
+    {
+        private List<string> gitPaths;
+        private List<string> placeholderFolderPaths;
+        private List<string> placeholderFilePaths;
+        private List<string> modifiedFolderPaths;
+        private List<string> modifiedFilePaths;
+        private int trackedFilesCount;
+        private int placeholderCount;
+        private int modifiedPathsCount;
+        private Dictionary<string, int> trackedFilesDirectoryTracking;

```

Comment:
I'm having some trouble understanding the names `trackedFilesDirectoryTracking` and `hydratedFilesDirectoryTracking`.  What is stored in these dictionaries?

Reply:
I renamed them to `gitTrackedFilesDirectoryTally` and `hydratedFilesDirectoryTally` respectively. They're used in the bottom part of the output which shows the most hydrated directories. They match the top level directories (key) to the number of hydrated paths (either placeholders or modified paths) which fall underneath them (value).